projects/canopy/src/lib/button/button.component.scss

```diff
@@ -1,4 +1,127 @@
-@import 'scss-modules/button';
```

Comment:
Nice one, I hated how I set this up! 👍 
Although I think we should mark it as a breaking change since these modules are used in some places (canopy landing page and possibly one more).

Reply:
I was wondering about that as a breaking change, probably better to be safe rather than sorry. I'll add some notes to the commit.

Reply:
tagged as a breaking change.